packages/css-library/src/stylesheets/formation-overrides/components/_forms.scss

```diff
@@ -11,11 +11,12 @@
 @use '../../override-function' as *;
 @use '../../mixins' as fm;
 
-$usa-form-width: scale-rem(32rem);
-
+$usa-form-width: 32rem;
```

Comment:
I think this value might have been updated unnecessarily. In formation, `$usa-form-width` is overridden in [this stylesheet](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/6bc6d59ae3fb119ac3ec1bb27995cd973301a70d/packages/formation/sass/base/_b-element-overrides.scss#L3), which is imported after all other imported stylesheets that involve form width. 

Here's a diff between local and staging that illustrates the side effect: 
![image](https://github.com/user-attachments/assets/bbd344c8-c3e4-425e-8712-6852285fc919)

And here's a diff with the scale-rem rule removed: 
![image](https://github.com/user-attachments/assets/fa2b88b8-1f39-4288-8521-e719623171c2)